app/services/chat.ts

```diff
@@ -17,6 +17,12 @@ import { assertIsDefined } from 'util/properties-type-guards';
 import * as remote from '@electron/remote';
 import { SourcesService } from 'app-services';
 
+/**
+ * Enables the twitch's Better TTV emotes
```

Comment:
Technically this function doesn't actually enable BTTV, it just returns the script that can be injected to enable it.  I think this comment could lead to confusion unless we update it to reflect that.

Reply:
Thanks for the feedback